src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search {
+  margin-bottom: 20px;
+  border-radius: 5px;
+  position: relative;
+}
+
+.search--big {
+  margin-top: 20px;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  display: block;
+  font-size: 16px;
+  color: #3d4e61;
+  line-height: 19px;
+  padding: 25px 70px 26px 62px;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 10%);
+  background: url(images/Search.svg) no-repeat;
```

Comment:
you can shorthand background and background-position
`background: url(images/Search.svg) no-repeat top 25px left 26px;`